examples/threathunter/threat_intelligence/results.py

```diff
@@ -58,9 +58,13 @@ def normalize(self):
 
         if self.severity <= 0 or self.severity > 10:
             logging.warning("normalizing OOB score: {}".format(self.severity))
-            self.severity = max(1, min(self.severity, 10))
-            # NOTE: min 1 and not 0
-            # else err 400 from cbapi: Report severity must be between 1 & 10
+            if self.severity > 10 and self.severity < 100:
+                #assume it's a percentage
+                self.severity = round(self.severity/10)
+            else:
+                self.severity = max(1, min(self.severity, 10))
+                # NOTE: min 1 and not 0
```

Comment:
I like the functionality of the else though it took a little to figure out what was going on. Would you mind updating the comment to describe this behavior? Something a long the lines that anything above 10 will be set to 10 and anything below 1 will be set to 1